src/spaces/utilities.jl

```diff
@@ -232,3 +232,11 @@ walk!(agent, ::typeof(rand), model::ABM{<:AbstractGridSpace{D}}; kwargs...) wher
 
 walk!(agent, ::typeof(rand), model::ABM{<:ContinuousSpace{D}}) where {D} =
     walk!(agent, Tuple(2.0 * rand(model.rng) - 1.0 for _ in 1:D), model)
+
+"""
+    spacesize(model::ABM)
+
+Return the size of the model's space. Works for [`AbstractGridSpace`](@ref) and
+[`ContinuousSpace`](@ref).
```

Comment:
Is there an easy way to make this also work for OSM space, which would return an ecnlosing rectangle with min max lon and min max lat?